ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/MultiDatePicker.java

```diff
@@ -0,0 +1,105 @@
+package net.sourceforge.ganttproject.calendar;
+
+import com.sun.javafx.scene.control.DatePickerContent;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableSet;
+import javafx.event.EventHandler;
+import javafx.scene.Node;
+import javafx.scene.control.DateCell;
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.skin.DatePickerSkin;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.TreeSet;
+
+public class MultiDatePicker extends DatePicker {
+
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    private ArrayList<LocalDate> selectedDates = new ArrayList();
+    private LocalDate lastSelectedDate = null;
+
+  public MultiDatePicker() {
```

Comment:
Please format this code with two space indents. There is an option in IDEA settings for the tab/indent size, and with this options set to 2 the default formatter should be okay.

 Or even better: translate it to Kotlin. There is an action for this too (Code > Convert Java File to Kotlin file in the IDEA menu)

Reply:
MultiDatePicker has been translated to Kotlin